test/e2e/join_and_leave_test.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package e2e
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+
+	placementv1beta1 "go.goms.io/fleet/apis/placement/v1beta1"
+)
+
+// Note that this container will run in parallel with other containers.
+var _ = Describe("placing wrapped resources using a CRP", Ordered, Serial, func() {
+	crpName := fmt.Sprintf(crpNameTemplate, GinkgoParallelProcess())
+	workNamespaceName := fmt.Sprintf(workNamespaceNameTemplate, GinkgoParallelProcess())
+	var wantSelectedResources []placementv1beta1.ResourceIdentifier
+	BeforeAll(func() {
+		// Create the test resources.
+		readEnvelopTestManifests()
+		wantSelectedResources = []placementv1beta1.ResourceIdentifier{
+			{
+				Kind:    "Namespace",
+				Name:    workNamespaceName,
+				Version: "v1",
+			},
+			{
+				Kind:      "ConfigMap",
+				Name:      testConfigMap.Name,
+				Version:   "v1",
+				Namespace: workNamespaceName,
+			},
+			{
+				Kind:      "ConfigMap",
+				Name:      testEnvelopConfigMap.Name,
+				Version:   "v1",
+				Namespace: workNamespaceName,
+			},
+		}
+	})
+
+	Context("Test cluster join and leave flow with CRP not deleted", Ordered, Serial, func() {
```

Comment:
```suggestion
	Context("Test cluster join and leave flow with CRP not deleted", func() {
```
nit, ordered and serial is redundant here as they're in parent already.